package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "vue-sfc-analyzer-webpack-plugin",
-  "version": "1.0.0-beta.2",
+  "version": "1.0.0",
   "description": "Webpack plugin for analyzing Vue SFC",
   "main": "dist/VueSFCAnalyzerWebpackPlugin.js",
   "typings": "dist/types/VueSFCAnalyzerWebpackPlugin.d.ts",
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=6.11.5"
```

Comment:
This Node version doesn't match the circleci configuration above.

Reply:
Picked the version of the minimum requirements of Webpack 4 😉 
CI's is because of our main env (and current maintained version).